streams/src/main/java/org/apache/kafka/streams/kstream/SessionWindows.java

```diff
@@ -134,14 +134,14 @@ public SessionWindows until(final long durationMs) throws IllegalArgumentExcepti
     }
 
     /**
-     * Reject late events that arrive more than {@code afterWindowEnd}
+     * Reject out-of-order events that arrive more than {@code afterWindowEnd}
      * after the end of its window.
-     *
+     * <p>
      * Note that new events may change the boundaries of session windows, so aggressive
-     * close times can lead to surprising results in which a too-late event is rejected and then
+     * close times can lead to surprising results in which an out-of-order event is rejected and then
```

Comment:
Similar to above, should we make use of the term "late" (out-of-order and arriving after the window is closed) to explain this phenomenon? I.e., I think our agreed definition of "late" is the same as what I previously used "too-late" for.